### PR TITLE
feat(app): landing page

### LINKED ISSUE #97: Create functional landing page
Instead of the placeholder we currently have, we should have the actual landing page!

See: https://www.figma.com/file/0ifTqGA2I3Y216eCJxh1rr/Keyman-Predictive-Text?node-id=1%3A2

Figma mockup of the landing page

### PR DESCRIPTION
**Pull request by eddieantonio**
[work in progress] attempts to render the mockup created by @pranavnarang.

Fixes #97

### CODE REVIEW

**Comment by eddieantonio** [comment]
It's not perfect, but have a look! I had difficulty with the wave graphic and the "workflow" image. I did something hacky for the wave, but left out the workflow; we can add in the workflow graphic later!

@pranavnarang, I'd appreciate your review, and a visual check on https://predictive-text-studio-git-feat-landing-page.eddieantonio.vercel.app/

**Review by jienius — APPROVED** [review]
This looks nice!

File: src/app/pages/LandingPage.svelte
Comment (pranavnarang) [inline src/app/pages/LandingPage.svelte:45]: Some more padding on the top would provide a little needed whitespace.


Current:
image

Figma:
image

File: src/app/pages/LandingPage.svelte
Comment (pranavnarang) [inline src/app/pages/LandingPage.svelte]: Can we replace the text with the Keyman logo if we have permission to use it?

`Keyman Logo`

File: src/app/pages/LandingPage.svelte
Comment (pranavnarang) [inline src/app/pages/LandingPage.svelte:103]: `margin-top` can be added here to add a little whitespace between the arrow and the quick start section.

File: src/app/pages/LandingPage.svelte
Comment (pranavnarang) [inline src/app/pages/LandingPage.svelte:191]: `margin-bottom` here would also provide a little needed whitespace to match designs

File: src/app/pages/LandingPage.svelte
Comment (pranavnarang) [inline src/app/pages/LandingPage.svelte:112]: The steps do seem a little close to each other, adding vertical margin space (top and bottom) would help. Another option would be to increase the height of the `quick-start` class and use flex box to dynamically space elements:

```css
.quick-start {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  align-items: center;
  height: x % / x rem / x px
}
```

**Comment by pranavnarang** [comment]
> It's not perfect, but have a look! I had difficulty with the wave graphic and the "workflow" image. I did something hacky for the wave, but left out the workflow; we can add in the workflow graphic later!
> 
> @pranavnarang, I'd appreciate your review, and a visual check on https://predictive-text-studio-git-feat-landing-page.eddieantonio.vercel.app/

Wave looks great! And the rest of the landing page is pretty close to the design, just a couple of places where extra spacing is needed in my opinion. 

We can recreate the workflow graphic using each individual asset (the keyboard, excel logo, google sheet logo, etc.) or just export the entire graphic as a png and place it in. While the latter is easier, it would probably cause issues with responsiveness.

File: src/app/pages/LandingPage.svelte
Comment (eddieantonio) [reply src/app/pages/LandingPage.svelte:112]: I used a margin between the `.quick-start__step`s ¯\\\_(ツ)\_/¯

**Comment by eddieantonio** [comment]
Thanks @pranavnarang for the comments! Could you please check out https://predictive-text-studio-git-feat-landing-page.eddieantonio.vercel.app/ one more time before we hit the merge button?

I think I'm gonna leave the workflow graphic for a future PR! 😅

**Review by pranavnarang — APPROVED** [review]
Looks great! Love the animation

**Pull request merged**
